### PR TITLE
What's new in the docs updates since early Sept

### PR DESCRIPTION
**Pull request by abbycross**
Both content and infra changes are reflected here.

### CODE REVIEW

**Comment by abbycross** [comment]
@Eric-Arellano I'd love some feedback on the wording of the infra stuff in here (under "user experience improvements") - thanks!

File: docs/guides/latest-updates.mdx [hunk 1]
```diff
@@ -144,41 +144,66 @@ From there, the blog goes on to deliver popular summaries of two recent arXiv pa
...
-- Tables in the docs now have borders to improve readability.
-- Alt-text has been added to all images to make them more accessible.
-- Heading size has been adjusted so that it does not change when the screen size changes.
-- Code styling has been improved, including bigger fonts and better contrast for code block comments.
-- In Search, you can now filter out terms by prefixing it with `-`, such as `-qiskit`
-- When on outdated versions of docs, the link in the top banner to take you to the latest version now preserves the page you were on.
+Updates to the infrastructure and design of the docs include the following:
+
+- We fixed GitHub source code links for functions using decorators - thanks to the external contributor who opened an issue for this!
+- When you click on "Go to the latest version" in the API reference docs, you arrive at the equivalent page in the latest version - and if the page does not exist in the new version, you land on the index page. We also fixed broken links in historical api refs
```
Comment (Eric-Arellano) [inline docs/guides/latest-updates.mdx:201]:
```suggestion
- When change the version in the API reference docs's left sidebar, you arrive at the equivalent page in the new version. If the page does not exist in the new version, you land on the index page.
- We fixed several broken links in historical API docs.
```

File: docs/guides/latest-updates.mdx [hunk 2]
```diff
@@ -144,41 +144,66 @@ From there, the blog goes on to deliver popular summaries of two recent arXiv pa
...
+- We improved the design and color of the warning banner for when you access outdated API pages to make it easier to realize that newer docs are available - and the banner persists as you scroll down the page
```
Comment (Eric-Arellano) [inline docs/guides/latest-updates.mdx:202]:
```suggestion
- We improved the design and color of the warning banner for when you access outdated API pages to make it easier to realize that newer docs are available. The banner also now persists as you scroll down the page.
```

File: docs/guides/latest-updates.mdx [hunk 3]
```diff
@@ -144,41 +144,66 @@ From there, the blog goes on to deliver popular summaries of two recent arXiv pa
...
+
+- Copyediting and typo fixes across the docs, including bugs reported by community contributors
 
 **New API reference docs**
 
-- [Qiskit SDK v1.2](../api/qiskit/release-notes/1.2)
-- [Qiskit Runtime v0.28](../api/qiskit-ibm-runtime/release-notes#0280-2024-08-15)
-- [qiskit-ibm-transpiler v0.5](../api/qiskit-ibm-transpiler/transpiler_service) 
+- [qiskit-ibm-runtime 0.29-0.31](/api/qiskit-ibm-runtime/release-notes)
+
```
Comment (Eric-Arellano) [inline docs/guides/latest-updates.mdx:192]:
```suggestion
```

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

**Pull request merged**